decomposed/extensions/MDDE_LDM/Profile/Model/Extended_Attributes/mdde_feature_business_rule_aggregate.xml

```diff
@@ -0,0 +1,9 @@
+<o:ExtendedAttributeTargetItem Id="52F0CC72-6B0C-44B8-A875-5DC31F6A52BA">
+<a:ObjectID>52F0CC72-6B0C-44B8-A875-5DC31F6A52BA</a:ObjectID>
+<a:Name>mdde_feature_business_rule_aggregate</a:Name>
+<a:CreationDate>1726842440</a:CreationDate>
+<a:Creator>RichaneRamsoedit</a:Creator>
+<a:Comment>This extended attribute is added to refer to the event handler CanCreate to be able to enable the aggregate business rule </a:Comment>
+<a:TargetCategory.Type>1</a:TargetCategory.Type>
+<a:Label>MDDE enable aggregate business rule</a:Label>
+</o:ExtendedAttributeTargetItem>
```

Comment:
Make sure there is a default on all these features set to 'True'. So for existing users there is no impact when updating to the latest extension.

Reply:
Filled in the default for the form